### PR TITLE
Feat :Added a tests to validate and update IFS historic interest star…

### PR DESCRIPTION
**Pull request by thadeusssewanyana**
…t - 6th Feb 2000 to 6 March 1999

### CODE REVIEW

File: src/test/resources/features/ifs/InterestRateChanges EdgeCases.feature [hunk 1]
```diff
@@ -155,6 +155,28 @@ Feature: Interest Rate Changes - Edge cases
...
+
+  Scenario: Interest start date - historic Interest date 1999-03-06
+    Given a debt item
+      | originalAmount | dateCreated | interestStartDate | interestRequestedTo | mainTrans | subTrans | interestBearing |
+      | 500000         | 1999-04-01  | 1999-03-06        | 2000-03-06          | 1525      | 1000     | true            |
+    And the debt item has no payment history
+    And no breathing spaces have been applied to the customer
+    And no post codes have been provided for the customer
+    When the debt item is sent to the ifs service
+    Then the ifs service wilL return a total debts summary of
```
Comment (sainahjan) [inline src/test/resources/features/ifs/InterestRateChanges EdgeCases.feature:167]: Typo?
```suggestion
    Then the ifs service will return a total debts summary of
```

**Review by sainahjan — APPROVED** [review]

**Review by sainahjan — APPROVED** [review]
